### PR TITLE
[REFERENT] Add ElectedRepresentative show page

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
[ch11413]

File: src/Entity/ElectedRepresentative/Mandate.php [hunk 1]
```diff
@@ -253,6 +277,26 @@ public function setNumber(int $number): void
...
+    {
+        if (!$this->politicalFunctions->contains($politicalFunction)) {
+            $this->politicalFunctions->add($politicalFunction);
+            $politicalFunction->setMandate($this);
+        }
+    }
+
+    public function removePoliticalFunction(PoliticalFunction $politicalFunction): void
+    {
+        if ($this->politicalFunctions->contains($politicalFunction)) {
```
Comment (ottaviano) [inline src/Entity/ElectedRepresentative/Mandate.php:290]: `removeElement` does the same check

File: templates/elected_representative/show.html.twig [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                    {{ elected_representative.age ? elected_representative.age ~ ' ans' : '~' }}
+                </div>
+                <div class="elected-representative-social-networks">
+                    {% for link in elected_representative.socialNetworkLinks %}
+                        <a target="_blank" href="{{ link.url }}" class="link--no-decor">
+                            <i class="fa fa-{{ link.type|lower }}{{ link.type is constant('App\\Entity\\ElectedRepresentative\\SocialLinkTypeEnum::INSTAGRAM') ? '' : '-square' }}"></i>
+                        </a>
+                    {% endfor %}
+                </div>
+                <div class="b__nudge--bottom-50">{{ elected_representative.isAdherent ? 'Adhérent' : '' }}</div>
```
Comment (kevin-verschaeve) [inline templates/elected_representative/show.html.twig:25]: the `: ''` part is not needed in twig

File: templates/elected_representative/show.html.twig [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                                </thead>
+                                <tbody>
+                                {% for sponsorship in elected_representative.sponsorships if sponsorship.candidate is not null %}
+                                    <tr class="referent__item">
+                                        <td>{{ sponsorship.candidate }}</td>
+                                        <td>{{ sponsorship.presidentialElectionYear }}</td>
+                                    </tr>
+                                {% else %}
+                                    <tr>
+                                        <td colspan="2" >
```
Comment (kevin-verschaeve) [inline templates/elected_representative/show.html.twig:48]: please remove useless space

File: templates/elected_representative/show.html.twig [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+                                <tbody class="no-border-bottom">
+                                {% for label in elected_representative.labels %}
+                                    <tr class="referent__item">
+                                        <td>{{ label.name }}</td>
+                                        <td>{{ label.beginYear }}</td>
+                                        <td>{{ label.finishYear }}</td>
+                                    </tr>
+                                {% else %}
+                                    <tr>
+                                        <td colspan="3" >
```
Comment (kevin-verschaeve) [inline templates/elected_representative/show.html.twig:77]: same

File: templates/elected_representative/show.html.twig [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+                                            {% if politicalFunction.beginAt %}
+                                                ({{ politicalFunction.beginAt|date('Y') }}{{ politicalFunction.finishAt ? '-' ~ politicalFunction.finishAt|date('Y') : '' }})
+                                            {% endif %}
+                                            <br />
+                                        {% endfor %}
+                                    </td>
+                                </tr>
+                            {% else %}
+                                <tr>
+                                    <td colspan="6" >
```
Comment (kevin-verschaeve) [inline templates/elected_representative/show.html.twig:124]: same

File: templates/elected_representative/show.html.twig [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+                            {% for mandate in elected_representative.notElectedMandates %}
+                                <tr class="referent__item">
+                                    <td>{{ mandate.name }}</td>
+                                    <td>{{ mandate.politicalAffiliation }}</td>
+                                    <td>{{ mandate.laREMSupport ? "elected_representative.mandate.larem_support.#{mandate.laREMSupport}"|trans : '' }}</td>
+                                    <td>{{ mandate.beginAt|localizeddate('none', 'none', 'fr_FR', null, 'd MMMM yyyy') }}</td>
+                                </tr>
+                            {% else %}
+                                <tr>
+                                    <td colspan="4" >
```
Comment (kevin-verschaeve) [inline templates/elected_representative/show.html.twig:152]: same

File: templates/elected_representative/show.html.twig [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+                                <th style="width: 50px;">Soutien LaREM</th>
+                                <th style="width: 50px;">Date</th>
+                            </tr>
+                            </thead>
+                            <tbody class="no-border-bottom">
+                            {% for mandate in elected_representative.notElectedMandates %}
+                                <tr class="referent__item">
+                                    <td>{{ mandate.name }}</td>
+                                    <td>{{ mandate.politicalAffiliation }}</td>
+                                    <td>{{ mandate.laREMSupport ? "elected_representative.mandate.larem_support.#{mandate.laREMSupport}"|trans : '' }}</td>
```
Comment (kevin-verschaeve) [inline templates/elected_representative/show.html.twig:147]: you can remove the `: ''`

Comment (kevin-verschaeve) [reply templates/elected_representative/show.html.twig:147]: actually you have more line with this, can you remove them all ?

**Pull request merged**